src/assets/locales/locale_config.js

```diff
@@ -2,6 +2,11 @@
 // For non "en" (since it's the default), use this mapping to define a fallback,
 // which will only be used if the specified locale file doesn't exist.
 
+export const AVAILABLE_LOCALES = {
+  en: "English",
+  zh: "简体中文"
+};
```

Comment:
Haha I didn't think about this before but I guess these don't need to be in translation files, do they :laughing: 
(I guess they _could_ be, as in "Spanish" vs "Español", but having them written in their own language ("Español", not "Spanish") seems better.)

Reply:
I was about to suggest adding flags, but this is sometimes discouraged: https://wplang.org/never-use-flags-language-selection/
This article _does_ recommend including _both_ the native language AND the translated name. I don't think it's necessary for right now (especially because the formatting of the dropdown isn't especially good), but something for us to consider.

Reply:
One nice thing about the flag is that lots of sites use something like a flag to indicate a language selection. I don't think it applies here because the preference screen is already "hidden" in the menu from the perspective of someone who doesn't read English (or w/e the default locale of the browser that we also support happens to be).
Makes me think we may want to elevate language selection to somewhere more easily noticed than the preferences screen (like something in the "settings menu" that "preferences" is in).

Reply:
You already use react-intl in the code base, you may want to use Language Display Names, see
https://formatjs.io/docs/intl#formatdisplayname
and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames
Chrome implements it. For Firefox you need the polyfill.